src/main.cpp

```diff
@@ -9,23 +9,29 @@
 #include "hitablelist.h"
 #include "camera.h"
 
+std::random_device d;
+std::mt19937 m{d()};
+std::uniform_real_distribution<float> dist(0.0f, 1.0f);
+
 Vec3 random_in_unit_sphere();
 
 Color ray_color(const Ray &r, Hitable *world);
 
 
 int main()
 {
-    Image i("test_diffuse.ppm");
+    Image image("test_diffuse.ppm");
 
     auto samples = 100;
 
-    Hitable *list[2];
+    Hitable *list[3];
 
-    list[0] = new Sphere(Vec3(0.0f, 0.0f, -1.0f), 0.5f);
-    list[1] = new Sphere(Vec3(0.0f, -5.0f, 0.0f), 5.0f);
+    int i = 0;
```

Comment:
Oh ok, that's why you turned `i` into `image`

Reply:
`int list_idx` was too long... :)